core/dnsserver/metrics.go

```diff
@@ -0,0 +1,14 @@
+package dnsserver
+
+import (
+	"github.com/coredns/coredns/plugin"
+	"github.com/prometheus/client_golang/prometheus"
+	"github.com/prometheus/client_golang/prometheus/promauto"
+)
+
+var httpsRequestCount = promauto.NewCounterVec(prometheus.CounterOpts{
+	Namespace: plugin.Namespace,
+	Subsystem: "https",
+	Name:      "requests_total",
+	Help:      "Counter of DNS over HTTPS requests.",
+}, []string{"server", "status"})
```

Comment:
I'm thinking this should go in `plugin/metrics/vars/vars.go`, where other server level metrics are declared.

Reply:
Moved & also renamed as it is more counter of responses per server & status code.